.github/workflows/get-translations.yml

```diff
@@ -31,17 +37,42 @@ jobs:
       - name: Fetch latest dev
         run: git fetch origin dev
 
-      - name: Get translations
-        run: npx ts-node -O '{"module":"commonjs"}' ./src/scripts/crowdin/translations/getTranslations.ts
-        env:
-          CROWDIN_API_KEY: ${{ secrets.CROWDIN_API_KEY }}
-          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
-
       - name: Authenticate GitHub CLI
         run: |
           echo ${{ secrets.GITHUB_TOKEN }} | gh auth login --with-token
 
+      # Build translations
+      - name: Build Crowdin project
+        id: build-crowdin
+        run: |
+          OUTPUT=$(npx ts-node -O '{"module":"commonjs"}' ./src/scripts/crowdin/translations/triggerBuild.ts)
+          echo "$OUTPUT"
+        shell: bash
+
+      - name: Await latest build to finish
+        env:
+          BUILD_ID: ${{ steps.build-crowdin.outputs.buildId}}
```

Comment:
This assigns the `buildId` output from the previous step  (step id: `build-crowdin`) to an environment variable to be accessed by the next script.